### PR TITLE
add debian packaging files

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

### CODE REVIEW

**Comment by lsm5** [comment]
@rhatdan @baude i'll try and have some debian packagers review this

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Priority: optional
+Maintainer: Lokesh Mandvekar <lsm5@fedoraproject.org>
+Build-Depends: debhelper (>=9), rsync, libassuan-dev,
+    golang, libglib2.0-dev, go-md2man, dh-golang,
+    golang-github-azure-go-ansiterm-dev,
+    libdevmapper-dev, libgpgme-dev, btrfs-progs
+Standards-Version: 3.9.6
+Homepage: https://github.com/projectatomic/buildah.git
+#Vcs-Git: git://anonscm.debian.org/collab-maint/cri-o.git
+#Vcs-Browser: https://anonscm.debian.org/cgit/collab-maint/cri-o.git
```
Comment (nalind) [inline debian/control]: These need to be updated.

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Package: buildah
+Architecture: all
+Depends: ${misc:Depends}
+Description: OCI-based implementation of Kubernetes Container Runtime Interface.
```
Comment (nalind) [inline debian/control]: That needs to be updated.

File: debian/copyright [hunk 3]
```diff
@@ -0,0 +1,38 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cri-o
```
Comment (nalind) [inline debian/copyright]: This needs to be updated.

File: debian/rules [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+#export DH_GOLANG_EXCLUDES := documentation vendor
+
+# Lots of shameless copy-paste from the debian package for prometheus
+
+BUILDDIR := $(shell pwd)
+
+%:
+	dh_clean
+	make clean
+	rm -rf $(BUILDDIR)/src $(BUILDDIR)/crio.conf
```
Comment (nalind) [inline debian/rules]: We don't include a crio.conf, so this needs to be updated.

File: debian/control (hunk 2 above)
Comment (lsm5) [inline debian/control]: ha, yup, copy paste from cri-o, will change it

File: debian/changelog [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+buildah (0.1-1ppa6) zesty; urgency=medium
+
+  * add dh-golang to build depends
+
+ -- Lokesh Mandvekar <lsm5@fedoraproject.org>  Tue, 27 Jun 2017 09:32:12 -0400
+
+buildah (0.1-1ppa5) zesty; urgency=medium
+
+  * remvoe git from build dependencies
```
Comment (TomSweeneyRedHat) [inline debian/changelog]: remvoe to remove

File: debian/control [hunk 6]
```diff
@@ -0,0 +1,17 @@
+Source: buildah
+Section: devel
+Priority: optional
+Maintainer: Lokesh Mandvekar <lsm5@fedoraproject.org>
+Build-Depends: debhelper (>=9), rsync, libassuan-dev,
```
Comment (TomSweeneyRedHat) [inline debian/control]: Are these suffcient?    In our README.md we list: make, golang (Requires version 1.8.1 or higher.), bats, btrfs-progs-devel, device-mapper-devel, gpgme-devel, libassuan-devel, git, bzip2, go-md2man, runc, and skopeo-containers.  I see some of these in the list below, but not all as I would expect.  But I've not done debian packaging in the past so I could be off base.

**Comment by lsm5** [comment]
I'd like to let this PR rest a bit, atleast until I've figured out a painless way to consistently build packages and send to PPA.

ATM the PPA process expects tarballs to be of the $NAME_$VERSION.orig.tar.gz format and will reject uploads if a tarball has the same name but with different contents. Which means I've yet to figure out how to build random intermediate commits where the version number doesn't change.

**Comment by lsm5** [comment]
closing, will open a new one asap, thanks for the comments guys

**Pull request closed without merging**